cobbler/actions/buildiso/netboot.py

```diff
@@ -546,11 +546,11 @@ def _generate_profile_config(
         if distro is None:
             raise ValueError("Distro of a Profile must not be None!")
         distroname = self.make_shorter(distro.name)
-        data = utils.blender(self.api, False, distro)
-        # SUSE uses 'textmode' instead of 'text'
-        utils.kopts_overwrite(
-            data["kernel_options"], self.api.settings().server, distro.breed
-        )
+        data = utils.blender(self.api, False, profile)
+        if distro is not None:  # SUSE uses 'textmode' instead of 'text'
```

Comment:
This check should not be needed because four lines above this was checked. Even type checkers should understand that?

Reply:
Right! Thanks for noticing. Fixed.